takoyaki/src/main/java/frc/robot/subsystems/Camera.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.subsystems;
+
+import java.util.List;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonUtils;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.math.util.Units;
+
+
+public class Camera {
+  private PhotonCamera camera;
+  private String name;
+  public Camera(String name) {
+    this.camera = new PhotonCamera(name);
+    this.name = name;
+  }
+
+  public void periodic() {
+    SmartDashboard.putBoolean("Camera \"" + name + "\" Sees Targets", camera.getLatestResult().hasTargets());
+  }
+  
+  public boolean hasTargets() {
+    return camera.getLatestResult().hasTargets();
```

Comment:
I think we should just get the latest result and then do all the below methods on that result, instead of calling `getLatestResult` every time we want to interact with the camera. imo it's fine to just interact directly with the `PhotonCamera` in the `CameraWithPID` subsystem, no need to write a wrapper class

Reply:
allowing the subsystem to talk directly with the `PhotonCamera` will also make the code a lot more readable and succinct, so you won't need to declare new exception types and do exception handling, etc.